src/main/java/fruit/shop/strategy/ActivityHandler.java

```diff
@@ -0,0 +1,5 @@
+package fruit.shop.strategy;
+
+public interface ActivityHandler {
+    int getOptionResult(int currentValue, int addValue);
+}

```

Comment:
Handler should do operation and save in Storage
```suggestion
public interface OperationHandler {
    void handle(FruitTransaction transaction);
}
```